### PR TITLE
Governance Engine OMAS - Refactoring

### PR DESCRIPTION
**Pull request by danielaotelea**

### CODE REVIEW

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/pom.xml [hunk 1]
```diff
@@ -75,11 +75,10 @@
             <artifactId>governance-engine-api</artifactId>
         </dependency>
 
-        <!-- Spring for calling REST APIs -->
-
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-core</artifactId>
+            <version>5.2.4.RELEASE</version>
```
Comment (planetf1) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-server/pom.xml]: This version identification needs removing -- versions should only be in dependencyManagement in our top level pom

Comment (danielaotelea) [reply open-metadata-implementation/access-services/governance-engine/governance-engine-server/pom.xml]: thanks! I removed it in the last commit.

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/publisher/GovernanceEnginePublisher.java [hunk 2]
```diff
@@ -2,187 +2,49 @@
...
+    private String eventToString(GovernanceEngineEvent engineEvent) {
+        ObjectMapper mapper = new ObjectMapper();
 
         try {
-            governanceEngineEventProcessor.processReclassifiedEntity(entity);
-        } catch (EntityProxyOnlyException | RepositoryErrorException | InvalidParameterException | UserNotAuthorizedException | PagingErrorException | TypeDefNotKnownException | EntityNotKnownException | PropertyErrorException | FunctionNotSupportedException | TypeErrorException e) {
-            log.debug("Governance Engine OMAS is unable to process a Re-Classified Entity event");
+            return mapper.writeValueAsString(engineEvent);
+        } catch (JsonProcessingException e) {
+            log.debug("[Governance Engine] Unable to map the event {} to string.", engineEvent);
```
Comment (bogdan-sava) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/publisher/GovernanceEnginePublisher.java]: debug log is OK but e Exception should be logged with error level or thrown

File: open-metadata-implementation/governance-servers/security-sync-services/security-sync-services-server/src/main/java/org/odpi/openmetadata/securitysyncservices/processor/SecuritySyncEventProcessor.java [hunk 3]
```diff
@@ -168,7 +168,7 @@ private GovernedAssetListAPIResponse getGovernedAssets() {
         try {
 
             ResponseEntity<String> result = restTemplate.exchange(governanceEngineURL, HttpMethod.GET, entity, String.class);
-            return (GovernedAssetListAPIResponse) mapToObject(result, GovernedAssetListAPIResponse.class);
+            return (GovernedAssetListResponse) mapToObject(result, GovernedAssetListResponse.class);
         } catch (HttpStatusCodeException exception) {
             log.debug("Unable to get the governed assets!");
```
Comment (bogdan-sava) [inline open-metadata-implementation/governance-servers/security-sync-services/security-sync-services-server/src/main/java/org/odpi/openmetadata/securitysyncservices/processor/SecuritySyncEventProcessor.java]: same here, exception should be logged or thrown
debug message might  not be shown and is not very helpful without more info about request

**Pull request merged**